src/main/java/io/github/sefiraat/slimetinker/events/InteractionEvents.java

```diff
@@ -159,7 +159,7 @@ public static void plateRefinedIron(EventFriend friend) {
             // We use helmet to control CD. You HAVE to have all four so using one stops 4 triggers
             Player p = friend.getPlayer();
             ItemStack i = friend.getHelmet();
-            Validate.notNull(i, "Helmet doesn't exist but has 4 stacks? I call hax");
+            Preconditions.checkNotNull(i, "Helmet doesn't exist but has 4 stacks? I call hax");
```

Comment:
`Preconditions.checkNotNull` will throw NPE instead of InvalidArgumentException, if you dont care the exception type, you can ignore this.

Reply:
YEah this is fine, but ty, didn't know that